Source/Charts/Charts/PieChartView.swift

```diff
@@ -134,15 +136,15 @@ open class PieChartView: PieRadarChartViewBase
         
         let radius = diameter / 2.0
         
-        let c = self.centerOffsets
-        
-        let shift = (data as? PieChartData)?.dataSet?.selectionShift ?? 0.0
+        var c = self.centerOffsets
+        c.x = radius + extraLeftOffset
         
         // create the circle box that will contain the pie-chart (the bounds of the pie-chart)
-        _circleBox.origin.x = (c.x - radius) + shift
-        _circleBox.origin.y = (c.y - radius) + shift
-        _circleBox.size.width = diameter - shift * 2.0
-        _circleBox.size.height = diameter - shift * 2.0
+        _circleBox.origin.x = (c.x - radius)
```

Comment:
Since you're providing different math for the `circleBox`, please provide screen shots of the before and after and a brief reasoning why the math needs to change.